cpg_infra/driver.py

```diff
@@ -631,6 +631,11 @@ def _build_config(values: list) -> str:
             # nest in .infrastructure
             d = {'infrastructure': dict(zip(keys, values))}
 
+            # JohnM 10 Dec 2024: Temporarily override QoB JAR to avoid ballooning hail costs
+            # cc7d0f is 0.2.132 with retry-all-400-errors (SET-261) and no 20million fixes (SET-247)
+            v132_retry_no20million = 'cc7d0fa48bd88b10b8c1c7315d9604ee855693e7'
```

Comment:
I really like how I understand what this variable is hahah

Reply:
The variable is only there because it wanted to reformat the long line when the constant was written out in the line below… 😂

Reply:
Great variable naming for sure